src/Tools/Source/RunTests/ProcessTestExecutor.cs

```diff
@@ -25,52 +26,68 @@ internal ProcessTestExecutor(TestExecutionOptions options)
             Options = options;
         }
 
-        public string GetCommandLine(AssemblyInfo assemblyInfo)
-        {
-            return $"{Options.XunitPath} {GetCommandLineArguments(assemblyInfo)}";
-        }
-
         public string GetCommandLineArguments(AssemblyInfo assemblyInfo)
         {
             var assemblyName = Path.GetFileName(assemblyInfo.AssemblyPath);
-            var resultsFilePath = GetResultsFilePath(assemblyInfo);
-            var xmlResultsFilePath = Path.ChangeExtension(resultsFilePath, ".xml");
-            var htmlResultsFilePath = Path.ChangeExtension(resultsFilePath, ".html");
 
             var builder = new StringBuilder();
-            builder.AppendFormat(@"""{0}""", assemblyInfo.AssemblyPath);
-            builder.AppendFormat(@" {0}", assemblyInfo.ExtraArguments);
-            builder.AppendFormat($@" -xml ""{xmlResultsFilePath}""");
+            builder.Append($@"test");
```

Comment:
might be worth commenting that this is about coming up with arguments to `dotnet` CLI in order to run this partition's tests, I had to blink at this for a sec to remember why we were passing the bare string `"test"` as a command line argument.